Tests/Unit/LoRaTools/RegionTest.cs

```diff
@@ -84,10 +90,25 @@ public void TestDownstreamRX2Frequency(Region region, double? nwksrvrx2freq, dou
             Assert.Equal(expectedFreq, freq);
         }
 
+        [Theory]
+        [MemberData(nameof(RegionEU868TestData.TestDownstreamRX2DataRateData), MemberType = typeof(RegionEU868TestData))]
+        [MemberData(nameof(RegionUS915TestData.TestDownstreamRX2DataRateData), MemberType = typeof(RegionUS915TestData))]
+        [MemberData(nameof(RegionCN470RP1TestData.TestDownstreamRX2DataRateData), MemberType = typeof(RegionCN470RP1TestData))]
+        [MemberData(nameof(RegionCN470RP2TestData.TestDownstreamRX2DataRateData), MemberType = typeof(RegionCN470RP2TestData))]
+        [MemberData(nameof(RegionAS923TestData.TestDownstreamRX2DataRateData), MemberType = typeof(RegionAS923TestData))]
+        public void TestDownstreamRX2DataRate(Region region, ushort? nwksrvrx2dr, ushort? rx2drfromtwins, ushort expectedDr, int? reportedJoinChannel = null, int? desiredJoinChannel = null)
```

Comment:
Again, this was deleted by accident in PR https://github.com/Azure/iotedge-lorawan-starterkit/commit/e2b1b7bac49c7cf9a474c50ddef6158c150d05d7#diff-4762bf50d3ee4a29e4ae67282a6cac6512e69662af8215f0278229ea9864b794; this is just bringing it back.